api/src/main/java/org/apache/iceberg/Table.java

```diff
@@ -201,11 +201,16 @@ default AppendFiles newFastAppend() {
 
   /**
    * Create a new {@link Rollback rollback API} to roll back to a previous snapshot and commit.
-   *
    * @return a new {@link Rollback}
    */
   Rollback rollback();
 
+  /**
+   * Create a new {@link ManageSnapshots API} to manage snapshots in this table and commit.
```

Comment:
Should be `{@link ManageSnapshots manage snapshots API}`. If you use additional text in a `@link` then only that text is shown, so this would read "Create a new API to manage snapshots . . ." and we want it to be "Create a new manage snapshots API".